### PR TITLE
old projects added July 2024

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

File: projects/MSU_Jacobson.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Department: Earth & Environmental Sciences
+Description: 'My students and I are planetary scientists who study the planets, moons,  asteroids, and comets of the Solar System as well as other planetary systems with  computational tools from the fields of celestial mechanics, geophysics, and geochemistry.  Our science is driven by big questions: where did our world come from? What other  kinds of worlds are out there? And, how unique is our planet''s history? (www.planetarymakerspace.org)'
```
Comment (brianhlin) [inline projects/MSU_Jacobson.yaml:2]:
```suggestion
Description: >-
  My students and I are planetary scientists who study the planets,
  moons, asteroids, and comets of the Solar System as well as other
  planetary systems with computational tools from the fields of
  celestial mechanics, geophysics, and geochemistry.  Our science is
  driven by big questions: where did our world come from? What other
  kinds of worlds are out there? And, how unique is our planets
  history? (www.planetarymakerspace.org)
```

File: projects/NYCDSS_Staff.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Department: Human Resources Administration
+Description: Anomaly detection research across various cohorts based on 
+user activity
+  modeling.
```
Comment (brianhlin) [inline projects/NYCDSS_Staff.yaml:4]:
```suggestion
Description: >-
  Anomaly detection research across various cohorts based on user
  activity modeling.
```

File: projects/TU_OTS.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Department: Office of Technology Services
+Description: Group for OTS staff at Towson University to learn about the 
+OSPool
```
Comment (brianhlin) [inline projects/TU_OTS.yaml:3]:
```suggestion
Description: Group for OTS staff at Towson University to learn about the OSPool
```

File: projects/UCSD_Andrijauskas.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Department: San Diego Supercomputing Center
+Description: 'Research students working with Fabio. Projects include: 
+information
+  visualization for the OSDF transfer logs, solar image processing using 
+the OSDF
+  and OSPool.'
```
Comment (brianhlin) [inline projects/UCSD_Andrijauskas.yaml:6]:
```suggestion
Description: >-
  Research students working with Fabio. Projects include: information
  visualization for the OSDF transfer logs, solar image processing
  using the OSDF and OSPool.
```

File: projects/UCSD_Sfiligoi.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Department: San Diego Supercomputing Center
+Description: The mission of the Global Infrastructure Lab (GIL) is to 
+tests and evaluate
+  infrastructure software. This project is used to exercise the OSPool 
+services.
```
Comment (brianhlin) [inline projects/UCSD_Sfiligoi.yaml:5]:
```suggestion
Description: >-
  The mission of the Global Infrastructure Lab (GIL) is to tests and
  evaluate infrastructure software. This project is used to exercise
  the OSPool services.
```

**Review by brianhlin — APPROVED** [review]
Fixing up some YAML formatting

@ChristinaLK did you generate these all with the form?

File: projects/UConn_Chen.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+Description: Bayesian methods for leveraging historical data in clinical trials and Bayesian adaptive designs. 
+Department: Statistics
+FieldOfScience: Statistics
+Organization: University of Connecticut
```
Comment (brianhlin) [inline projects/UConn_Chen.yaml:4]:
```suggestion
InstitutionID:  https://osg-htc.org/iid/eq81k8qpbcq9 
Organization: University of Connecticut
```

**Comment by brianhlin** [comment]
@opensciencegrid/project-office do you approve the following new institutions?

- New York City Department of Social Services
- Towson University

If so, please pass along their IDs

**Comment by ChristinaLK** [comment]
I did use the form, but I also used nano on my laptop which is doing weird line wrapping. :P

**Comment by mwestphall** [comment]
@opensciencegrid/project-office following up on this, are we OK'd to add the following two institutions?
- New York City Department of Social Services
- Towson University

**Comment by jstathas** [comment]
*   New York City Department of Social Services
  *   Towson University
are both approved.

Janet




From: mwestphall ***@***.***>
Date: Monday, July 22, 2024 at 9:08 AM
To: opensciencegrid/topology ***@***.***>
Cc: Stathas, Janet ***@***.***>, Team mention ***@***.***>
Subject: Re: [opensciencegrid/topology] old projects added July 2024 (PR #4008)

@opensciencegrid/project-office<https://github.com/orgs/opensciencegrid/teams/project-office> following up on this, are we OK'd to add the following two institutions?

  *   New York City Department of Social Services
  *   Towson University

File: projects/NYCDSS_Staff.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Department: Human Resources Administration
+Description: >-
+  Anomaly detection research across various cohorts based on user
+  activity modeling.
+FieldOfScience: Computer and Information Science and Engineering
+FieldOfScienceID: '11.0101'
+InstitutionID: Unknown
```
Comment (mwestphall) [inline projects/NYCDSS_Staff.yaml:7]:
```suggestion
InstitutionID: https://osg-htc.org/iid/sg7tvfh0b232
```

File: projects/TU_OTS.yaml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+Department: Office of Technology Services
+Description: Group for OTS staff at Towson University to learn about the OSPool
+FieldOfScience: Computer and Information Services
+FieldOfScienceID: '11.1099'
+InstitutionID: Unknown
```
Comment (mwestphall) [inline projects/TU_OTS.yaml:5]:
```suggestion
InstitutionID: https://osg-htc.org/iid/jthubnuqeush
```

**Pull request merged**